brainstormforce/astra
# AST-2806 - Core blocks padding, margin & other css issues fixed to match frontend

### Description

Table → 

Table caption alignment and font size in different in editor and frontend  - [Annotation on 2023-01-03 at 12-41-51.png](https://d.pr/i/9Qxqog)

 Spacing in between table and caption is different in editor and frontend.

Table border is different in editor than frontend : editor - [Annotation on 2023-01-03 at 12-39-53.png](https://d.pr/i/rzYGcb) and frontend - [Annotation on 2023-01-03 at 12-40-03.png](https://d.pr/i/kXcryc)

Verse → 

Verse block size difference : editor - [Annotation on 2023-01-03 at 13-01-44.png](https://d.pr/i/nBTOtZ) and frontend - [Annotation on 2023-01-03 at 13-01-52.png](https://d.pr/i/CmvaY0)

Preformatted → 

Different size : editor - [Home • Droplr](https://d.pr/i/vEKdM0,) frontend - [Annotation on 2023-01-03 at 13-11-35.png](https://d.pr/i/WIeCJX)

Search → 

Border is different in editor and frontend : editor - [Annotation on 2023-01-03 at 13-42-24.png](https://d.pr/i/l8n6yL) and frontend - [Annotation on 2023-01-03 at 13-41-46.png](https://d.pr/i/gV3JNH)

Padding right and left should be 10px - [Annotation on 2023-01-03 at 13-44-14.png](https://d.pr/i/6K0Xh4)

Separator → 

Separator is different in editor and on frontend : editor - [Annotation on 2023-01-03 at 12-47-57.png](https://d.pr/i/cuQ8Cf) and frontend - [Annotation on 2023-01-03 at 12-48-05.png](https://d.pr/i/v0dVzo)

JIRA - https://brainstormforce.atlassian.net/browse/AST-2806

### Screenshots

Mentioned in description

### Types of changes


- Block editor core blocks css fixes to match frontend.
- Core blocks with reported issues: Verse, Preformatted, Table, Search



### How has this been tested?

- Create Verse, Table, Search, Preformatted block in editor.
- Verse block size should be same as frontend.
- Preformatted block size should be same as frontend.
- Table caption alignment and font size should be same in editor & frontend.
- Search block border should be same in editor & frontend.
- Search block border should apply same as frontend when border color is changed from customizer.
- Search block padding left & right should be same.
- Seperator block should look same as frontend.

### Checklist:
